scripts/docker_build.sh

```diff
@@ -1,6 +1,7 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
What Bash feature is being used now?

Reply:
Why was this resolved while being unanswered?

Reply:
Is already fixed.

Reply:
If so, then not in the correct commit. See how the current https://github.com/oss-review-toolkit/ort/pull/5760/commits/02787e58c054c34fe1ca07f2017131e0c2347212 still changes this line.

Reply:
Yeah, you're changing it back only in https://github.com/oss-review-toolkit/ort/pull/5760/commits/07c68b664a7d2b9b1d7c367e4812feadafe8d677. What a mess...

Reply:
I have fixed it now the "docker_build: Enable builds on Apple Silicon" no longer changes line 1.